app/Http/Controllers/Api/AuthController.php

```diff
@@ -24,6 +38,66 @@ public function register(StorePostRequest $request)
 
         $token = $user->createToken('Token Name')->accessToken;
 
-        return  response()->json(['success' => true, 'token' => $token], 201);
+        return  response()->json(['success' => true, 'token' => $token], ResponseAlias::HTTP_CREATED);
+    }
+
+    public function login(LoginRequest $request)
+    {
+        $credentials = $request->validated();
+
+        if (Auth::attempt($credentials)) {
+            $user = User::where('email', $credentials['email'])->first();
+
+            $token = $user->createToken('Token Name')->accessToken;
+
+            return response()->json(['status' => ResponseAlias::HTTP_OK, 'token' => $token], ResponseAlias::HTTP_OK);
+        }
+
+        return response()->json(['message' => 'Wrong credentials'], ResponseAlias::HTTP_UNPROCESSABLE_ENTITY);
+    }
+
+    public function sendResetPasswordEmail(ResetPasswordRequest $request)
+    {
+        $email = $request->validated();
```

Comment:
'validated' function returns all validated data from request. In this case you get array and $email isn't correct name for this variable. Name it: $requestData or $data. $email name confuses